gravity.lisp

```diff
@@ -0,0 +1,70 @@
+;; Copyright (C) 2016 Joram Schrijver
```

Comment:
I moved it to a separate file because it's being used in various places, not message-window specific, and because I was going to be using it in StumpUI. After everything has been ported over to StumpUI, I think this should be moved into there.